source/selinux/selinux.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package selinux
+
+import (
+	"io/ioutil"
+	"github.com/golang/glog"
+)
+
+type Source struct{}
+
+func (s Source) Name() string { return "selinux" }
+
+func (s Source) Discover() ([]string, error) {
+	features := []string{}
+	status, err := ioutil.ReadFile("/sys/fs/selinux/enforce")
+	if err != nil {
+		glog.Info("Failed to detect the status of selinux, please check if the system supports selinux and make sure /sys is mounted into the container: %s", err)
```

Comment:
You need to return here similar to https://github.com/kubernetes-incubator/node-feature-discovery/blob/master/source/pstate/pstate.go#L37.